### PR TITLE
Update Weights and Biases Integration

### PR DESCRIPTION
**Pull request by soumik12345**
## Describe changes
Updating [Weights and Biases](https://wandb.ai/) Integration.

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/features/integrations) table.
- [ ] I have added tests to cover my changes.

## Types of changes
New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by htahir1** [comment]
@soumik12345 Would you mind signing the CLA? https://cla-assistant.io/zenml-io/zenml?pullRequest=486

**Comment by soumik12345** [comment]
> @soumik12345 Would you mind signing the CLA? https://cla-assistant.io/zenml-io/zenml?pullRequest=486

Hi @htahir1 , I have signed the CLA

File: pyproject.toml [hunk 1]
```diff
@@ -61,6 +61,8 @@ distro = "^1.6.0"
 rich = {extras = ["jupyter"], version = "^11.0.0"}
 httplib2 = "<0.20,>=0.19.1"
 pyparsing = "<3,>=2.4.0"
+wandb = "^0.12.12"
```
Comment (htahir1) [inline pyproject.toml:64]: Hey @soumik12345 , we dont usually add dependencies in our core dependency project.toml. I would ask you to remove changes to the pyproject.toml and put them instead in `src/zenml/ntegrations/wandb/__init__.py`. We have another way of controlling these dependencies

**Review by htahir1 — CHANGES_REQUESTED** [review]
Thanks for signing the CLA! Just one small comment and then we can merge and move forward!

**Comment by htahir1** [comment]
Thank you so much for the help!

**Pull request merged**